cis_v140/section_6.sp

```diff
@@ -0,0 +1,81 @@
+locals {
+  cis_v140_6_common_tags = merge(local.cis_v140_common_tags, {
+    cis_section_id = "6"
+  })
+}
+
+benchmark "cis_v140_6" {
+  title         = "6 Storage"
+  documentation = file("./cis_v140/docs/cis_v140_6.md")
+  children = [
+    control.cis_v140_6_1,
+    control.cis_v140_6_2,
+    control.cis_v140_6_3,
+    control.cis_v140_6_4
+  ]
+
+  tags = merge(local.cis_v140_6_common_tags, {
+    type    = "Benchmark"
+    service = "Azure/ActiveDirectory"
+  })
+}
+
+control "cis_v140_6_1" {
+  title         = "6.1 Ensure document sharing is being controlled by domains with whitelist or blacklist"
+  description   = "You should control sharing of documents to external domains by either blocking domains or only allowing sharing with specific named domains."
+  sql           = query.office365_domain_sharing_restriction_mode.sql
+  documentation = file("./cis_v140/docs/cis_v140_6_1.md")
+
+  tags = merge(local.cis_v140_6_common_tags, {
+    cis_item_id = "6.1"
+    cis_level   = "2"
+    cis_type    = "automated"
+    license     = "E3"
+    service     = "Azure/ActiveDirectory"
```

Comment:
Should the service still be `Azure/ActiveDirectory`, since this is about Office 365?